package.json

```diff
@@ -2,10 +2,7 @@
   "name": "electronim",
   "version": "0.0.0",
   "description": "Free/Libre open source Electron based multi instant messaging (IM) client.",
-  "author": {
-    "name": "Marc Nuri",
-    "url": "https://www.marcnuri.com"
-  },
+  "author": "Marc Nuri (https://www.marcnuri.com)",
```

Comment:
Please restore the changes to the package.json or send **only** the one for the `directories` entry in a different commit.

Reply:
Noted. I will work on this ASAP

Reply:
I have made the necessary changes to the package.json file, as well as the formatting on the index.js and index.test.js files